hotshot-state-prover/src/service.rs

```diff
@@ -382,19 +354,34 @@ pub async fn sync_state<ApiVer: StaticVersionType>(
     tracing::debug!("Old state: {old_state:?}");
     tracing::debug!("New state: {:?}", bundle.state);
 
+    let blocks_per_epoch = config
```

Comment:
Is the `epoch_height` changing? If not we should initialize it only once.
cc @bfish713 @ss-es @jbearer

Reply:
yup, i asked the exact question on zulip, and it sounds like this won't change and will only be used as genesis setup. iiuc.

Reply:
so you want to put it inside `StateProverConfig`? that's the only place to persist. 
imo, it's not a big deal to query it on demand. shouldn't be too costly. but yeah, open to suggestions here.